CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ endif()
 
 include_directories(${PROJECT_SOURCE_DIR})
 
+# find kokkos
+find_package(Kokkos 3 REQUIRED)
```

Comment:
I think calling `find_package(Cabana)` should now incl. finding `kokkos` as well.

Reply:
It looks like if I just link `Cabana::cabanacore` then I get ` Target "minipic" links to target "Kokkos::kokkos" but the target was not` even though Kokkos is not on the link line

Reply:
That would be a bug in our export target, `Cabana::cabanacore` should through `Kokkos::kokkos` in automatically.

Reply:
Any short term plans to fix in Cabana? I can keep my `find_pacakge(Kokkos..` for now